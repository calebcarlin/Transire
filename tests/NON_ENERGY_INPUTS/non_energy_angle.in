#=== Interface options
crys_a_file = graphene.xyz
crys_a_surface = 0 0 1
crys_a_layers = 1
crys_b_surface = 0 0 1
crys_b_layers = 1
crys_b_file = graphene.xyz
separation = 0.3
tolerance = 1.0e-1
calculate_initial_energy = False
project_name = non_energy_angle
max_atoms = 100000
output_file = non_energy_angle.out
print_debug = True
surface_search = False
#=== Search options
search_list = 1

#======= Specific Angle options (search 1)
angles_stepsize = 10
number_of_angles = 7
starting_angle = -10.0
angle_calculate_energy = False
write_coord_file = True
