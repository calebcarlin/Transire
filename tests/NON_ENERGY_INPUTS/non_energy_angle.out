Too many atoms, skipping to next step
Too many atoms, skipping to next step
Too many atoms, skipping to next step
Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

Beginning twist angle search

Surface Indices : (0, 0, 1)/(0, 0, 1)

===========

beginning rotation of 0.0 degrees:

printing output: angle = 0.0 atoms = 8

===========

beginning rotation of 10.0 degrees:

printing output: angle = 10.0 atoms = 116

===========

beginning rotation of 20.0 degrees:

printing output: angle = 20.0 atoms = 212

===========

beginning rotation of 30.0 degrees:

Too many atoms, skipping to next step
===========

beginning rotation of 40.0 degrees:

printing output: angle = 40.0 atoms = 244

===========

beginning rotation of 50.0 degrees:

printing output: angle = 50.0 atoms = 172

===========

beginning rotation of 60.0 degrees:

printing output: angle = 60.0 atoms = 8


Lowest Energy configuration correspond to angle 60.0

